components/chart/events.md

```diff
@@ -222,6 +223,10 @@ The `OnLegendItemClick` event fires when the user clicks on any item in the Char
 
 The `OnSeriesClick` event fires as a response to the user click on a `<ChartSeries>`.
 
+## OnDrilldown
```

Comment:
This new content is injected before and after information that is for `OnSeriesClick`. Better move it up.